releases/release-7.4.0.md

```diff
@@ -0,0 +1,186 @@
+---
+title: TiDB 7.4.0 Release Notes
+summary: Learn about the new features, compatibility changes, improvements, and bug fixes in TiDB 7.4.0.
+---
+
+# TiDB 7.4.0 Release Notes
+
+Release date: xx xx, 2023
+
+TiDB version: 7.4.0
+
+Quick access: [Quick start](https://docs.pingcap.com/tidb/v7.4/quick-start-with-tidb) | [Installation packages](https://www.pingcap.com/download/?version=v7.4.0#version-list)
+
+7.4.0 introduces the following key features and improvements:
+
+<!-- key feature placeholder-->
+
+## Feature details
+
+### Performance
+
+- note 1
+
+- note 2
```

Comment:
```suggestion
* Introduce cloud-based global sorting capability to improve the performance and stability of parallel execution tasks such as `ADD INDEX` or `IMPORT INTO` [#45719](https://github.com/pingcap/tidb/issues/45719) @[wjhuang2016](https://github.com/wjhuang2016) **tw@ran-huang** <!--1456-->

    Before v7.4.0, when executing distributed parallel execution framework tasks like `ADD INDEX` or `IMPORT INTO`, TiDB nodes need to prepare a large local disk for sorting encoded index KV pairs and table data KV pairs. Due to the inability to sort from a global perspective, there might be overlapping data between different TiDB nodes and within each node during import. This would cause TiKV to constantly perform compaction operations when importing these KV pairs into TiKV, reducing the performance and stability of `ADD INDEX` or `IMPORT INTO`.

    With the introduction of the global sorting feature in v7.4.0, the encoded data is now written to cloud storage instead of being written locally and sorted. Global sorting is performed in cloud storage, and then the indexed data and table data that have undergone global sorting are imported into TiKV in parallel, thereby improving performance and stability.

    For details, see [user document](/tidb-global-sort.md)。
```